src/tests/lockup.spec.ts

```diff
@@ -20,10 +21,32 @@ test("view all info about lockup account on 'finality' block", async (t) => {
 });
 
 test("view all info about lockup account on particular block", async (t) => {
+  const arhivalConfig = {
+    nodeUrl: "https://archival-rpc.mainnet.near.org",
+    networkId: "mainnet",
+    keyStore: connectOptions.keyStore,
+    headers: connectOptions.headers
+  };
+
   const res = await viewLockupAccount(
     lockupDataMock[1].lockupAccountId,
-    null,
+    arhivalConfig,
     { block_id: 59976515 }
   );
   t.deepEqual(res, lockupDataMock[1]);
 });
+
+test("view all info about account on particular block using arhival credencials", async (t) => {
+  const arhivalConfig = {
+    nodeUrl: "https://archival-rpc.mainnet.near.org",
+    networkId: "mainnet",
+    keyStore: connectOptions.keyStore,
+    headers: connectOptions.headers
+  };
+  const res = await viewLockupAccount(
+    lockupDataMock[2].lockupAccountId,
```

Comment:
> mock objects are simulated objects that mimic the behavior of real objects in controlled ways

What you have here is not a mock, it is a snapshot. I suggest we use `lockupAccountSnapshots` name here.